samples/maps/demo/us-data-labels/demo.js

```diff
@@ -60,9 +60,10 @@
                 format: '{value}%'
             },
             stops: [
-                [0.2, '#188e2a'], // Green
-                [0.6, '#fee401'], // Yellow
-                [1, '#df1309'] // Red
+                [0.0, '#440154'],
+                [0.33, '#3b528b'],
+                [0.66, '#21908d'],
+                [1.0, '#fde725']
```

Comment:
Full color-map should be implemented to ensure accurate presentation. See [here](https://www.kennethmoreland.com/color-advice/) for color stops, and alternative color axis.

```suggestion
                [0.00, '#440154'],
                [0.14, '#46327f'],
                [0.29, '#365c8d'],
                [0.43, '#277f8e'],
                [0.57, '#1fa187'],
                [0.71, '#4ac26e'],
                [0.86, '#9fda3a'],
                [1.00, '#fde725']
```